runtime/v1/shim/service.go

```diff
@@ -120,18 +120,8 @@ type Service struct {
 
 // Create a new initial process and container with the underlying OCI runtime
 func (s *Service) Create(ctx context.Context, r *shimapi.CreateTaskRequest) (_ *shimapi.CreateTaskResponse, err error) {
-	var mounts []process.Mount
```

Comment:
Not sure if relevant / adding more context (I'm trying to get my head around some of that patch); this was added in https://github.com/containerd/containerd/commit/ea00fc793bd1dca2e944bcd1fc8b576fe995c753 / https://github.com/containerd/containerd/pull/2091)